src/app/pages/FeedPage/components/Twit.tsx

```diff
@@ -6,6 +6,43 @@ import SvgRetwit from 'app/components/SVG/SvgRetwit'
 import { Twit as TwitTypes } from 'types/Twit'
 import Avatar from 'app/components/Avatar'
 
+const elapsedDate = (time: Date) => {
+  const date = new Date(
+    (time || '')
```

Comment:
Throw error if `time` not defined or simply not date. Also there are no need to call construct of date again.